Dockerfile.megamolbart

```diff
@@ -1,12 +1,12 @@
 # Copyright 2020 NVIDIA Corporation
-FROM nvcr.io/nvidia/nemo:1.2.0
+FROM nvcr.io/nvidia/nemo:1.3.0
```

Comment:
Was NeMo version bumped due to a security issue? I'm doubtful the model code that's currently on main of the MegaMolBART repo will work with NeMo 1.3.0, but of course it will change soon anyway with the model parallel model. Just wanted to note this.

Reply:
NeMo 1.2 is pointing to a Nvidia package repo that is not accessible. This switch is made mainly to ensure we can build the container. I did not notice any functional issue after the change.